chimney/src/main/scala/io/scalaland/chimney/internal/macros/MappingMacros.scala

```diff
@@ -54,8 +54,15 @@ trait MappingMacros extends Model with TransformerConfiguration {
         }
         val wasRenamed = lookupName != target.name
         fromGetters
-          .find(lookupAccessor(config, lookupName, wasRenamed, From))
-          .map(ms => ResolvedAccessor(ms, wasRenamed))
+          .map(lookupAccessor(config, lookupName, wasRenamed, From))
+          .sortBy {
```

Comment:
Why sorting is needed here? Can't we simplify it to finding first resolution that is other than `NotFound`? Like:

```scala
fromGetters
  .find(ms => lookupAccessor(config, lookupName, wasRenamed, From)(ms) != AccessorResolution.NotFound)
  .getOrElse(AccessorResolution.NotFound)
```

Reply:
my intention was to take `AccessorResolution.Resolved` over `AccessorResolution.DefAvailable`.  Not sure if it's possible to have `val` and method which fit to some target field

Reply:
My thesis is that you can't have both `Resolved` and `DefAvailable` for given `fromGetter` list. If that's true, you can safely eliminate sorting. But I might be wrong. The easiest way to be sure is to add a test that ensures this.